app/controllers/UserController.py

```diff
@@ -0,0 +1,196 @@
+from app.models.user import User
```

Comment:
Hi @MrV173 please rename this file to `user_controller` instead

![Screenshot 2023-12-26 at 07 54 29](https://github.com/project-zusammen/ioc-backend/assets/46950501/d60021e7-884f-4008-911a-7656a9bfb59d)


as discussed here:
https://softwareengineering.stackexchange.com/questions/308972/python-file-naming-convention#:~:text=modules%20(filenames)%20should%20have%20short,should%20use%20the%20CapWords%20convention.

Reply:
done